src/pages/studyView/charts/pieChart/PieChart.tsx

```diff
@@ -207,6 +208,13 @@ export default class PieChart extends React.Component<IPieChartProps, {}>
             : d.count.toLocaleString();
     }
 
+    @computed
```

Comment:
it doesn't make sense to make this a computed property because it's returning a callback which will be executed outside the scope of mobx's optimization.  you should make this a bound method of this class instead of a computed getter.  the cost of deriving the color for a handful of piechart slices is negligible.

Reply:
the importance of this comment is mostly just to make sure you understand what the computed decorator does.  you can read about it in mobx documentation if necessary.

Reply:
I looked at line 135 fill() methods using @computed as well, it also returns a method but not a value. Should I change its tag to @autobind as well? I've tried to change either fill() or labelColor() and got "@boundMethod decorator can only be applied to methods not: undefined"